src/utils/fetchClient.ts

```diff
@@ -1,39 +1,41 @@
 const BASE_URL = 'https://mate.academy/students-api';
 
-// a promise resolved after a given delay
 function wait(delay: number) {
   return new Promise(resolve => {
     setTimeout(resolve, delay);
   });
 }
 
-// To have autocompletion and avoid mistypes
 type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
 
 function request<T>(
   url: string,
   method: RequestMethod = 'GET',
-  data: any = null, // we can send any data to the server
+  data: unknown = null,
```

Comment:
why did you change it?

Reply:
because of typescript, it wouldnt let me get away with the any type
))